core/src/test/java/org/polypheny/db/snapshot/MockRelSnapshot.java

```diff
@@ -78,7 +78,7 @@ public MockRelSnapshot( MockSnapshot snapshot ) {
 
     @Override
     public @NonNull Optional<LogicalTable> getTable( String namespaceName, String tableName ) {
-        Optional<MockTable> table = snapshot.tables.stream().filter( t -> t.name.equals( tableName ) ).findFirst();
+        Optional<MockTable> table = snapshot.tables.stream().filter( t -> t.name.equalsIgnoreCase( tableName ) ).findFirst();
```

Comment:
I know, this is only a mock class for testing, but shouldn't the case-sensetivity be defined by the namespace?

Reply:
Yes and no, I am hesitant to make those classes too clever. There main intend should be to provide a simple way to mock structure. They should not emulate the Snapshot in internal logic as this tends to populate the complexity of mock. I would like to leave them as barebones and simple as possible. But I will annotate that it is always case insensitive.

Reply:
Ok, makes sense